containers/Makefile

```diff
@@ -152,6 +180,15 @@ build/repositories/concent/: build/package-builder/clone-or-update.sh /var/conce
 build/repositories/golem/: build/package-builder/clone-or-update.sh /var/concent-data/golem
 	build/package-builder/clone-or-update.sh  "golem" "$@"
 
+$(SIGNING_SERVICE_SOURCE_FROM_CONCENT_API): build/signing-service/%: build/concent-api/%
+	mkdir --parents "build/verifier/"
+	cp "$<" "$@"
+
+build/signing-service/signing-service/: build/repositories/concent/ build/concent-api/concent-api.tar
+	cp --recursive "$<signing_service/." "$@"
+	cp "$<RELEASE-VERSION" "$@"
```

Comment:
Please remove the `/` from the end of the first prerequisite so that in commands you can use `$</RELEASE-VERSION`. Otherwise the `$<` variable looks pretty confusing.

Reply:
Outdated after changing the solution.